statick_tool/plugins/tool/cccc_tool_plugin.py

```diff
@@ -56,13 +58,21 @@ def scan(  # pylint: disable=too-many-branches
             cccc_config = self.plugin_context.args.cccc_config
         config_file = self.plugin_context.resources.get_file(cccc_config)
         if config_file is not None:
-            opts = "--opt_infile=" + config_file
+            opts = ["--opt_infile=" + config_file]
         else:
             return []
+        opts.append(" --lang=c++")
+
+        issues: List[Issue] = []
 
         for src in package["c_src"]:
+            tool_output_dir: str = ".cccc-" + Path(src).name
+            temp_opts = opts
+            temp_opts.append("--outdir=" + tool_output_dir)
```

Comment:
I tested locally and don't see the need for the temporary variable here. Could be

```suggestion
            opts.append("--outdir=" + tool_output_dir)
```

Then change `temp_opts` to `opts` in `subproc_args`.